nvm.sh

```diff
@@ -1711,6 +1711,7 @@ nvm_print_versions() {
     -v installed_color="$INSTALLED_COLOR" -v system_color="$SYSTEM_COLOR" \
     -v current_color="$CURRENT_COLOR" -v default_color="$DEFAULT_COLOR" \
     -v old_lts_color="$DEFAULT_COLOR" -v has_colors="$NVM_HAS_COLORS" '
+function alen(arr, i, len) { len=0; for(i in arr) len++; return len; }
```

Comment:
this is bash-specific syntax that won't work in dash/sh, and i suspect its definition could survive an `nvm deactivate`. is there any way to avoid adding a new shell function to the scope?

Reply:
Not really, that's an awk function, not for bash/sh.

Reply:
ahhh ok, it's defined in the awk context, thanks